tests/Tests.AzureAppConfiguration/JsonContentTypeTests.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Azure;
+using Azure.Data.AppConfiguration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.AzureKeyVault;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.JsonConfiguration;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Xunit;
+
+namespace Tests.AzureAppConfiguration
+{
+    public class JsonContentTypeTests
+    {
+        List<ConfigurationSetting> _kvCollection = new List<ConfigurationSetting>();
```

Comment:
>List<ConfigurationSetting> _kvCollection = new List<ConfigurationSetting>(); [](start = 8, length = 76)

Since I see most test cases are clearing this collection in the beginning and populating it, how about creating it inside each test instead?

Reply:
Done

---
In reply to: [454634233](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/181#discussion_r454634233) [](ancestors = 454634233)